.editorconfig

```diff
@@ -18,5 +18,5 @@ indent_size = 4
 [*.{css,html}]
 indent_size = 2
 
-[*.md]
+[*.{md,py}]
 trim_trailing_whitespace = false
```

Comment:
Not actually needed, but there where some python files that had trailing spaces. I thought if black accepted those python files, then trailing spaces are good.
This can be removed in a different commit/PR. Because I do not want to add more noise to this PR